package.json

```diff
@@ -7,6 +7,7 @@
     "npm": "~3.8.x"
   },
   "dependencies": {
+    "app": "^0.1.0",
```

Comment:
hmmm weird - ok I was just following errors that are fixed at https://codeclimate.com/github/18F/identity-idp/pull/1363 -- should we disable instead?